RoverReachImage:
    ImageProcessing:
        pub_pxCoordinates: '/px_coordinates'
        pub_pxCoordinates1: '/px_coordinates1'
        resize: False #false recommended for quality of image
        resize_width: 1200 #only works when resize true
        resize_height: 800 #only woerks when resize true
        direction: '/artag_direction'


    CalculateBearing:
        sub_pxCoordinates: '/px_coordinates'
        sub_pxCoordinates1: '/px_coordinates1'
        pub_bearing: '/bearing_to_ball'
        camAngleOfView: 110
